reasoner/resolution/ResolverRegistry.java

```diff
@@ -188,6 +191,16 @@ public ResolverView registerResolvable(Resolvable<?> resolvable) {
         return ResolverView.retrievable(resolver, retrievable.retrieves());
     }
 
+    public Actor.Driver<BoundRetrievableResolver> registerBoundRetrievable(Retrievable retrievable, ConceptMap bounds) {
```

Comment:
We need an elegant way to terminate these

Reply:
Ah yes, currently we don't terminate the bound resolvers when we're actually done with them, because they are bound to a root and iteration.